app/components/Browser/Browser.jsx

```diff
@@ -149,9 +149,12 @@ export default class Browser extends Component
             updateTab,
             activeTabBackwards,
             activeTabForwards,
+
```

Comment:
Are these empty lines intentional, shall we add comments to separate them by categories/types if that's the intention?

Reply:
Yeh they were intentional, so yeh, I can encomment 👍